pylint/checkers/imports.py

```diff
@@ -189,15 +189,15 @@ def _dependencies_graph(filename, dep_info):
     for depmodname, dependencies in sorted(dep_info.items()):
         for modname in dependencies:
             printer.emit_edge(modname, depmodname)
-    printer.generate(filename)
+    return printer.generate(filename)
```

Comment:
Can you upgrade (create 😄) the typing here, please ?

Reply:
Sorry, I don't understand what you mean.